docs/glossary.md

```diff
@@ -11,3 +11,22 @@
 - **Game**: (AKA Game XML) refers to a specific XML within a map
 - **Map**: collection of files, eg: images, polygons text, properties, and at least one game XML
 
+## Battle Terms:
+- **Roll**: How many dice can the unit roll.
+- **Strength**: The number on the dice that determines whether a hit was made. Any number less than or equal to the number is a hit.
+- **Round**: A battle round includes both sides firing at each other. A battle can consist of multiple rounds. During and at the end of each round, casualties are removed from the game and can not participate anymore.
+- **Phase**: A battle phase is subpart of a battle round where specific rules are in play that determine what the units can do. Some phases allow the units to retreat. Other phases allow the units to fire at each other.
```

Comment:
nit, perhaps fully qualify this as a 'Battle Phase', to disambiguate from a 'turn phase'. In other words, a 'phase' can be multiple things, so I think we are being specific to a 'battle phase' here.